src/PrestaShopBundle/Controller/Admin/Improve/International/CurrencyController.php

```diff
@@ -500,6 +565,14 @@ private function getErrorMessages(Exception $e)
                     'This currency already exists.',
                     'Admin.International.Notification'
                 ),
+                CurrencyConstraintException::EMPTY_BULK_TOGGLE => $this->trans(
+                    'Currencies must be provided in order to toggle their status',
+                    'Admin.International.Notification'
+                ),
+                CurrencyConstraintException::EMPTY_BULK_DELETE => $this->trans(
+                    'At least one currency must be provided for deleting',
```

Comment:
@LouiseBonnard 'Can't find the domain of these 2 strings.
The rest of the wording is ok!

Reply:
See https://github.com/PrestaShop/PrestaShop/pull/18883#discussion_r431121110 please

Reply:
I'm waiting for confirmation from @jolelievre (or someone else) to update this my PR

Reply:
These are new wording, that's why you can't find them The question is/are do they need modification? Or are there existing wording that can be used? (I couldn't find some)
And, lastly, what domain should they be in?

Reply:
@jolelievre  See #18883 (comment) please !

Reply:
I didn't see Louise's response But I think we can go for `You must select at least one element to perform a bulk action.` (only one new wording this way, and reusable in many cases)
It only leaves us with the domain `Admin.International.Notification` or `Admin.Notifications.Error` Since it's a generic one maybe the second one is better

Reply:
Yep sorry @PululuK I saw it 30sec after ^^ I think you can make the modification with `You must select at least one element to perform a bulk action.` and `Admin.Notifications.Error`
As it has high chances to be validated, and we will just have to wait for wording validation

Thanks again for your commitment and patience ;)

Reply:
> Thanks again for your commitment and patience ;)

Don't worry @jolelievre  I am a poet... not a web developer !